test/src/search/search_navigation_test.dart

```diff
@@ -262,6 +262,41 @@ d
         null,
       );
     });
+
+    testWidgets('Changing search pattern with match at current '
+    'selection will not jump to next', (wt) async {
```

Comment:
If you are testing that something is NOT happening, put the test to `issues` directory. Because most of the things the world can do to us do NOT happen, and it is unclear why we should test this one.

If keeping the test here, name it after what SHOULD happen.